great_expectations/core/_docs_decorators.py

```diff
@@ -1,6 +1,10 @@
+from __future__ import annotations
+
 from textwrap import dedent
 from typing import Any, Callable, TypeVar
 
+from great_expectations import _DYNAMIC_DEFINITIONS
+
```

Comment:
Why are we importing this? Why not just define this global in `_docs_decorators.py`?

Reply:
I was just trying to find a place for it that wouldn't result in circular imports. If I put it back there I will have the same problem.